dmsite/venv/src/Authentication/LoginTextField.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Created by christinadaley on 1/18/19.
+ */
+
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import {
+  Checkbox,
+  Form,
+  Icon,
+  Input,
+  Button,
+  Spin,
+} from 'antd';
+
+import * as actions from '../store/actions/auth';
+
+const antSpinner = <Icon type="loading" style={{ fontSize: 24 }} spin />;
+
+class TextField extends React.Component {
+    handleSubmit = (e) => {
+      const { form, onAuth } = this.props;
+      e.preventDefault();
+      form.validateFields((err, values) => {
+        if (!err) {
+          onAuth(values.email, values.password);
+          console.log('Received values of form: ', values);
```

Comment:
Maybe use `console.debug` for more organization. I don't think its actually any different, but it'll let you sort by debug message when debugging.